test/vstest.console.UnitTests/Processors/TestAdapterPathArgumentProcessorTests.cs

```diff
@@ -160,71 +160,121 @@ public void InitializeShouldUpdateTestAdapterPathInRunSettings()
         [TestMethod]
         public void InitializeShouldMergeTestAdapterPathsInRunSettings()
         {
-            var runSettingsXml = "<RunSettings><RunConfiguration><TestAdaptersPaths>d:\\users;f:\\users</TestAdaptersPaths></RunConfiguration></RunSettings>";
-            var runSettings = new RunSettings();
-            runSettings.LoadSettingsXml(runSettingsXml);
-            RunSettingsManager.Instance.SetActiveRunSettings(runSettings);
-            var mockFileHelper = new Mock<IFileHelper>();
-            var mockOutput = new Mock<IOutput>();
+            var currentActiveSetting = RunSettingsManager.Instance.ActiveRunSettings;
```

Comment:
Shouldn't the `TestInitialize` and `TestCleanup` handle such primitives?

Reply:
Yes, this can be achieve using TestInitialize and TestCleanup but here only couple of tests need that. should we go via TestInitialize and TestCleanup path?

Reply:
Fixed!!